ETC.tsv

```diff
@@ -14174,7 +14174,7 @@ ETC_20150804_014175	Until the light device at Loading Place gets activated
 ETC_20150804_014176	Among the remains of the collapse, the light device of the loading place has been discovered.{nl}The light device is being activated!
 ETC_20150804_014177	The light device at loading place of manufacturing factory has been activated!{nl}The light comes into the loading place of the Mausoleum
 ETC_20150804_014178	Because of the unstable structure of the loading place at Mausoleum{nl}the collapse occured at the manufacturing company
-ETC_20150804_014179	Wendigo appeared out of the gap!
+ETC_20150804_014179	A Wendigo jumped out of the aperture!
```

Comment:
I assume an 'aperture' is an opening/gap? I googled the term since I have never heard of it and it showed me an app. 
The term is only mentioned here and not in the dialogue and quest tracker. 
It's a fancy word and others might get confused about it.